### PR TITLE
Adding project rtbid

### PR DESCRIPTION
**Pull request by ooovinia**
Adding project rtbid.

### CODE REVIEW

File: files/rtbid/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "ooovinia"
+github = "https://github.com/ooovinia"
+homepage = "http://github.com/"
```
Comment (megawac) [inline files/rtbid/info.ini]: can you fix these links to the repository page

File: files/rtbid/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "rtbid",
+  "repo": "ooovinia/rtbid",
```
Comment (megawac) [inline files/rtbid/update.json]: this repo doesn't exist

File: files/rtbid/info.ini (hunk 1 above)
Comment (ooovinia) [inline files/rtbid/info.ini]: now is it rigth?

**Comment by jimaek** [comment]
I dont understand what you are trying to do. Where is the actual project? Your links are all wrong

**Pull request closed without merging**